src/Microsoft.Health.Dicom.Tests.Common/AcceptHeaderHelpers.cs

```diff
@@ -47,13 +47,20 @@ public static AcceptHeader CreateAcceptHeaderForGetFrame(string transferSyntax =
           quality: quality);
     }
 
-    public static AcceptHeader CreateRenderJpegAcceptHeader(string transferSyntax = "*", string mediaType = KnownContentTypes.ImageJpeg, double? quality = null, PayloadTypes payloadType = PayloadTypes.SinglePart)
+    public static AcceptHeader CreateRenderJpegAcceptHeader(string transferSyntax = "*", string mediaType = KnownContentTypes.ImageJpeg, PayloadTypes payloadType = PayloadTypes.SinglePart)
     {
         return CreateAcceptHeader(
           transferSyntax: transferSyntax,
           payloadType: payloadType,
-          mediaType: mediaType,
-          quality: quality);
+          mediaType: mediaType);
```

Comment:
Since mediaType is a param, do we need 2 methods?

Reply:
removed the second